app.js

```diff
@@ -56,6 +60,29 @@ app.use(async (ctx, next) => {
     await next();
 });
 
+async function verifyAccountOwnership({ ctx, accountId, securityCode, signature }) {
+    // ensure securityCode is a recent block
+    const givenBlock = Number(securityCode);
+    if (Number.isNaN(givenBlock)) ctx.throw(403);
+
+    const currentBlock = (await ctx.near.connection.provider.status()).sync_info.latest_block_height;
```

Comment:
Is there a better way to get current block height?

Reply:
Don't think so